lib/components/sitewide/TitleBar.vue

```diff
@@ -33,6 +33,8 @@ v-toolbar(app)
         network-manager
 
     help-menu
+    v-btn(flat icon @click="setDarkMode(!darkMode)" :class=" {'toggle-up': darkMode, 'toggle-dn':!darkMode}")
```

Comment:
Great button, icon, and animation choice! I'd fix the inconsistent spacing in the class declaration:
- unnecessary space `" {`
- missing space between `:!`